foundation-extensions/experience-platform-extension/edge-network-api-reference.md

```diff
@@ -279,6 +296,187 @@ public class EdgeEventHandle {
 ```
 
 Use this class when calling the sendEvent API with EdgeCallback.
+
+#### ExperienceEvent
+
+Experience Event is the event to be sent to Adobe Experience Edge Network.
+The XDM data is required for any Experience Event being sent using the Edge extension.
+
+```Java
+public final class ExperienceEvent {
```

Comment:
I'm wondering if here, instead of giving the source to the class, we can just provide simple examples for each API.  The IdentityMap API reference does something similar where it gives examples of initializing an object and example of calling each API.
https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network/api-reference#identitymap

Reply:
Added examples to gitbook.